var/spack/repos/builtin/packages/savanna/package.py

```diff
@@ -43,7 +43,7 @@ class Savanna(MakefilePackage):
 
     depends_on('mpi')
     depends_on('stc')
-    depends_on('adios +fortran +sz +staging')
+    depends_on('adios +fortran +staging')
```

Comment:
Please do not remove this change. Packages should **never** rely on default variants. The default may change from time-to-time, and users may override the defaults in `packages.yaml` or on the command-line. If `savanna` only works with `adios+sz`, you have to make that explicit.